### ISSUE TITLE
Download Video link does not work? 

### ISSUE THREAD

**Issue by ilovefood2** [download-online-video/chrome-avgle-helper#10]
updated to new version, and when Download video shows up, clicking on it does not do anything, known issue? 

![image](https://user-images.githubusercontent.com/4293869/59700412-3d058b80-91c1-11e9-948a-3c800f0de7f3.png)

**Comment by vide0** [comment]
Are you using branch `3.x.x`?

I haven't pushed the build codes to here. Because I am testing it. You can build codes by yourself.

``` bash
npm install
npm run build
```
And put windows library files into `%userprofile%\bin` if you are Windows user.

**Comment by ilovefood2** [comment]
my bad, I thought it was 3.xx

im Mac user btw

**Comment by ilovefood2** [comment]
ok after updating to 3.0.0,  clicking on Download link will actually download a .sh file, then have to  run sudo sh filename.sh 
and then it gets to errors

`  -> [HttpSkipResponseCommand.cc:240] errorCode=22 The response status is not successful. status=400`

I randomly tried a couple videos, same error and sudo is asked each time. before it didn't ask for sudo

**Comment by vide0** [comment]
Actually, `3.x.x` is developing, I haven't tested it on Mac. But it doesn't need sudo permission. you can run it via command `bash ./download-xxxxx.sh` or `zsh ./download-xxx.sh`. And you can paste the content of log file in temporary directory  to here if you meet other download problem.

**Comment by vide0** [comment]
By the way, The original download commands still could be use. And I will push the latest code of branch `3.x.x` to here later, you can watch and pull it.

**Comment by vide0** [comment]
@ilovefood2 And can you help to check what happen if you change the `download-xxx.sh` to `download-xx.command` and click it in Finder

**Comment by ilovefood2** [comment]
yes original command on top of video webpage still works, if copying it and run in bash, it works fine. but downloaded.sh is asking for sudo to run and even with sudo, gives error too. 

changed to .command then it still won't run, by default, it's normal, the downloaded file on mac does not have executable property. so I had to chmod +x file.command, then clicking on it will run then.

actually, just remove .sh extension and add x property will get it run, it's just more manual steps to do compared with ver 2.x.x

btw I am on macOS 10.14.5

**Comment by zhangchen91** [comment]
下载中途失败了怎么重试，保留已下载的ts文件不变继续下载新的？
